herbert_core/classes/MPIFramework/@parallel_config/parallel_config.m

```diff
@@ -491,7 +491,13 @@
         function obj = set.parallel_threads(obj,n_threads)
             n_threads = floor(n_threads);
             n_workers = get_or_restore_field(obj, 'parallel_workers_number');
-            n_poss_threads = floor(obj.n_cores/n_workers);
+            if ispc % let's assume pc is Intel so threades are double number of cores
```

Comment:
What do you mean "assume it's intel"? Do you mean it has hyperthreading? You should be careful
a) because hyperthreading my be either captured in `n_cores` or it may not. Sometimes you have hyperthreading mapped as a logical (but not a physical) processor other times it may just be a hidden feature handled internally as in some ARM systems. 
b) Because hyperthreading may not always be advantageous to use if you're already saturating the processors with work and can result in slowdown.